src/mots/utils.py

```diff
@@ -0,0 +1,59 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Utility helper functions."""
```

Comment:
This utilities can probably all be moved closer to their associated domains for better discoverability:

* `generate_machine_readable_name`: This might be better being beside the `class Module` definition.
* `get_list_input`: Chuck this in `cli.py` IMHO
* `parse_real_name`: This can be beside `class Person`

Reply:
I do like being able to separate and test these small methods that do not rely on any external classes in their own module (i.e. `utils`). I can see why they could be moved to a different module if they had any dependencies, but they're basically text parsers that may be used by any other module -  thus I think they're fine to stay here.